tests/instruments/test_all_instruments.py

```diff
@@ -70,6 +70,8 @@ def find_devices_in_module(module):
             # On Windows instruments.ni.daqmx can raise an OSError before ModuleNotFoundError
             # when checking installed driver files
             pass
+        except AttributeError:
+            pass
```

Comment:
This was missed as I have to add this because of the NI / nicau.dll issue for not having National Instruments drivers or visa installed on my PC.

This at least allows me to verify my git clone locally is running all of the tests correctly including the linting before I push the committed changes. I wish the NI instruments modules didn't punish people that don't use them and I think its a Python 3.11->3.12 change that causes the issue.

These two lines are removed.